tests/e2e/defaultvmgroup_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test tries to add, remove and replace vm to the _DEFAULT tenant.
+// Expected behavior is that add/rm/replace vm for _DEFAULT vmgroup should fail.
+
+package e2e
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	con "github.com/vmware/docker-volume-vsphere/tests/constants/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/misc"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	ErrorMsg = "This feature is not supported for vmgroup _DEFAULT."
+)
+
+type DefaultVmGroupTestSuite struct {
+	esxIP    string
+	hostName string
+}
+
+func (s *DefaultVmGroupTestSuite) SetUpTest(c *C) {
+	s.hostName = govc.RetrieveVMNameFromIP(os.Getenv("VM2"))
+	s.esxIP = os.Getenv("ESX")
+	out, err := admincli.ConfigInit(os.Getenv("ESX"))
+	c.Assert(err, IsNil, Commentf(misc.FormatOutput(out)))
+}
+
+func (s *DefaultVmGroupTestSuite) TearDownTest(c *C) {
+	out, err := admincli.ConfigRemove(s.esxIP)
+	c.Assert(err, IsNil, Commentf(misc.FormatOutput(out)))
+}
+
+var _ = Suite(&DefaultVmGroupTestSuite{})
+
+// Test steps:
+// 1. Add a vm to the _DEFAULT tenant
+// 2. Remove a vm from the _DEFAULT tenant
+// 3. Replace a vm from the _DEFAULT tenant
+func (s *DefaultVmGroupTestSuite) TestModifyDefaultTenantVMs(c *C) {
+	log.Printf("START: defaultvmgroup.TestModifyDefaultTenantVMs")
+	out, err := admincli.AddVmToVmgroup(s.esxIP, con.DefaultVmgroup, s.hostName)
```

Comment:
This is one option, but we might have other better options to avoid so many folders/packages just for constants. Let's think about this and revisit this part if any ideas coming to mind.